spec.md

```diff
@@ -95,61 +105,92 @@ Everything that applies to licenses in this section also applies to license
 
 Each Covered File MUST have Copyright and Licensing Information associated with
-it. There are two ways to associate Copyright and Licensing Information with a
-file. In addition, there is a way to associate Copyright and Licensing
-Information with a snippet.
+it. You can associate Copyright and Licensing Information with a file in the
+following ways:
+
+- Comment headers
+- REUSE.toml
+- DEP5 (deprecated)
+
+REUSE.toml and DEP5 are mutually exclusive. You MUST NOT use both
+simultaneously.
+
+Additionally, you can associate Copyright and Licensing Information with
+Snippets inside of files.
 
 ### Comment headers
 
-To implement this method, each Commentable File SHOULD
-contain comments at the top of the file (comment header) that declare that
-file's Copyright and Licensing Information.
+To implement this method, a Commentable File MUST declare the file's Copyright
+and Licensing Information in a comment header. The Information SHOULD be as
+close to the top of the top of the file as possible in a comment header. The
+Commentable File SHOULD use UTF-8 encoding.
 
 For Uncommentable Files, the comment header that declares the file's Copyright
-and Licensing Information SHOULD be in an adjacent UTF-8 encoded text file of the same 
-name with the additional extension `.license` (example: `cat.jpg.license` if the 
-original file is `cat.jpg`).
+and Licensing Information MUST be in an adjacent text file of the same name with
+the additional extension `.license` (example: `cat.jpg.license` if the original
+file is `cat.jpg`). The adjacent file SHOULD use UTF-8 encoding.
 
 `.license` files MAY be used with Commentable Files, but it is still RECOMMENDED
 that comment headers be put inside Commentable Files.
 
-The comment header MUST contain one or more `SPDX-FileCopyrightText` tags, and one or
-more `SPDX-License-Identifier` tags. A tag is followed by a colon, followed by
-a text value, and terminated by a newline.
-
-The `SPDX-FileCopyrightText` tag MUST be followed by a copyright notice.
-
-Instead of the `SPDX-FileCopyrightText` tag, the symbol `©`, or the word `Copyright` MAY
-be used, in which case a colon is not needed.
+The comment header MUST contain one or more Copyright Notices and one or more
+`SPDX-License-Identifier` tag-value pairs. A tag is followed by a colon,
+followed by a text value, and terminated by a newline.
 
 The `SPDX-License-Identifier` tag MUST be followed by a valid SPDX License
 Expression describing the licensing of the file (example:
-`SPDX-License-Identifier: GPL-3.0-or-later OR Apache-2.0`). If separate sections
-of the file are licensed differently, a different `SPDX-License-Identifier` tag
-MUST be included for each section.
+`SPDX-License-Identifier: GPL-3.0-or-later OR Apache-2.0`).
 
 An example of a comment header:
 
 ```
 # SPDX-FileCopyrightText: 2016, 2018-2019 Jane Doe <jane@example.com>
-# SPDX-FileCopyrightText: 2019 Example Company
+# SPDX-FileCopyrightText: 2019 Example NGO
```

Comment:
Sure, a minor change to promote the holding of copyright not by companies but by other entities. Perhaps a simple word 'organisation' is sufficient?
```suggestion
# SPDX-FileCopyrightText: 2019 Example Organisation
```

Reply:
ok!